client/components/mma/billing/Billing.tsx

```diff
@@ -296,6 +305,10 @@ function renderProductBillingInfo([mmaCategory, productDetails]: [
 	);
 }
 
+function renderInAppPurchase(value: AppSubscription) {
+	return <div key={value.subscriptionId}>todo</div>;
```

Comment:
This won't be visible, right? Just double checking the feature switch works

Reply:
Yes it's behind the feature switch on line 363